yt_dlp/extractor/nuum.py

```diff
@@ -0,0 +1,200 @@
+                'channel_id': ('media_container_channel', 'channel_id', {str_or_none}),
+            }),
+            **traverse_obj(stream, {
+                'view_count': ('stream_total_viewers', {int_or_none}),
+                'concurrent_view_count': ('stream_current_viewers', {int_or_none}),
+            }),
+            **traverse_obj(media, {
+                'duration': ('media_duration', {int_or_none}),
+                'thumbnail': ('media_meta', ('media_preview_archive_url', 'media_preview_url'), {url_or_none}),
+            }, get_all=False),
+        }
+
+
+class NuumMediaIE(NuumBaseIE):
+    IE_NAME = 'nuum:media'
+    _VALID_URL = r'https?://nuum\.ru/(?:streams|videos|clips)/(?P<id>[\d]+)'
+    _TESTS = [{
+        'url': 'https://nuum.ru/streams/1592713-7-days-to-die',
+        'only_matching': True,
+    }, {
+        'url': 'https://nuum.ru/videos/1567547-toxi-hurtz',
+        'md5': 'f1d9118a30403e32b702a204eb03aca3',
+        'info_dict': {
+            'id': '1567547',
+            'ext': 'mp4',
+            'title': 'Toxi$ - Hurtz',
+            'description': '',
+            'timestamp': 1702631651,
+            'upload_date': '20231215',
+            'thumbnail': r're:^https?://.+\.jpg',
+            'view_count': int,
+            'concurrent_view_count': int,
+            'channel_id': '6911',
+            'channel': 'toxis',
+            'duration': 116,
+        },
+    }, {
+        'url': 'https://nuum.ru/clips/1552564-pro-misu',
+        'md5': 'b248ae1565b1e55433188f11beeb0ca1',
+        'info_dict': {
+            'id': '1552564',
+            'ext': 'mp4',
+            'title': 'Про Мису 🙃',
+            'timestamp': 1701971828,
+            'upload_date': '20231207',
+            'thumbnail': r're:^https?://.+\.jpg',
+            'view_count': int,
+            'concurrent_view_count': int,
+            'channel_id': '3320',
+            'channel': 'Misalelik',
+            'duration': 41,
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        video_data = self._call_api(f'media-containers/{video_id}', video_id, 'media')
+
+        m3u8_url, info = self._parse_video_data(video_data)
+
+        formats, subtitles = self._extract_m3u8_formats_and_subtitles(
+            m3u8_url, video_id, 'mp4', live=info['is_live'])
+        info.update({
+            'formats': formats,
+            'subtitles': subtitles,
+        })
+        return info
+
+
+class NuumLiveIE(NuumBaseIE):
+    IE_NAME = 'nuum:live'
+    _VALID_URL = r'https?://nuum\.ru/channel/(?P<id>[^/#?]+)/?(?:$|[#?])'
+    _TESTS = [{
+        'url': 'https://nuum.ru/channel/mts_live',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        channel = self._match_id(url)
+        channel_info = self._get_channel_info(channel)
+        if traverse_obj(channel_info, ('channel', 'channel_is_live')) is False:
```

Comment:
I'd just do this since we don't want to continue extraction unless truthy
```suggestion
        if not traverse_obj(channel_info, ('channel', 'channel_is_live')):
```